keylime/tpm/tpm_main.py

```diff
@@ -670,7 +669,7 @@ def _parse_mb_bootlog(self, log_b64: str) -> typing.Tuple[Failure, typing.Option
 
     def parse_mb_bootlog(
         self, mb_measurement_list: Optional[str], hash_alg: algorithms.Hash
-    ) -> typing.Tuple[Dict[str, int], typing.Optional[Dict[str, List[str]]], RefState, Failure]:
+    ) -> typing.Tuple[Dict[str, int], typing.Optional[Dict[str, List[str]]], Dict[str, Any], Failure]:
```

Comment:
Why RefState is replaced here? Seems to be defined as `typing.Mapping[str, tests.Data]`

Reply:
I made a conscious decision to remove all types defined in `elchecking` from the rest of the code.

* Ultimately `refstate` should be redefined as "mba policy", because refstate is not even a term anyone in TCG would recognize; we made it up. 
* The type of what is now "refstate" should be eventually defined in `mba.py` and become part of the API.

I just didn't want to move *everything* at once, it would have made the PR even more unwieldy and complicated. My focus is on preserving functionality while we are slowly cleaning up.